frontend/package.json

```diff
@@ -25,6 +25,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "run": "^1.4.0",
```

Comment:
where is this used?

Reply:
It isn't (as far as I know), but I was having some problems with running the dev server without it.

But also, maybe I'm running the dev server incorrectly, which is one of the reasons I wanted to be sure to have you and @davemfish take a look at this.